cps/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -319,7 +319,7 @@ public File getStorageDir() throws IOException {
     public void start() throws IOException {
         final CpsScript s = parseScript();
         scriptClass = s.getClass();
-        s.initialize();
+        s.$initialize();
```

Comment:
Nope, it is legal. Generally use of it is discouraged in non-generated code. This is sort of a special case. I considered also `__initialize__` etc.